README.md

```diff
@@ -2,20 +2,20 @@
 
 # Project Overview
 
-## Setup
-Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
-
 ## Secrets File
-DEVEUI: which can be found by scanning the QR code on the LoRaWan chip. 
-APPEUI: currently we use the same APPEUI as the one for BraveButtons
-APPEUI: APPEUI = hex(str(hex(DEVEUI)) + str(hex(APPEUI)))
 An example one can be found in our private [living doc](https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-8442)
+- DEVEUI: which can be found by scanning the QR code on the LoRaWan chip. 
+- APPEUI: currently we use the same APPEUI as the one for BraveButtons
+- APPEUI: APPEUI = hex(str(hex(DEVEUI)) + str(hex(APPEUI))), where '+' is string concatenation
+
+## Setup
+Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
 
 ### Using Arduino Extension for VSCode
 I recommend using VSCode's Arduino extension for easier file management, after going through the setup process with the Arduino IDE at least once. After opening the project with VSCode, if you notice red squiggly error messages for missing includes, follow [this](https://stackoverflow.com/questions/52234438/vs-code-giving-header-errors-for-arduino-missing-official-header) guide on how to fix them. On Windows 10, the board firmware files can be found at `C:/Users/{user}/AppData/Local/Arduino15/packages/rak_rui/hardware/stm32/3.4.2/cores/STM32WLE/component/rui_v3_api`, and can be added under `includePath` of the `.vscode/c_cpp_properties.json` file. 
 
 ## Serial Debugging
-Can be done with any serial monitor (e.g. Arduino, Arduino's VSCode Extension, PlatformIO, etc.) with BAUD rate of 115200. 
+Can be done with any serial monitor (e.g. Arduino, Arduino's VSCode Extension, PlatformIO, etc.) with BAUD rate of 115200. Remember to compile the file with `DEBUG_LOG` set to `true`, or `DEBUG_LOG_MORE` set to `true` in `main.h` to be able to see serial logs. 
```

Comment:
Are there any negative consequences for leaving these values `TRUE` in production? If so, then I suggest a warning her and in the code to not commit the `TRUE` values.

Reply:
Should allow for reduced power consumption. I've added it.